pkgs/development/tools/rust/cargo-pgrx/buildPgrxExtension.nix

```diff
@@ -178,6 +178,11 @@ let
       (args.checkFeatures or [ ])
       ++ (lib.optionals usePgTestCheckFeature [ "pg_test" ])
       ++ [ "pg${pgrxPostgresMajor}" ];
+
+    meta = (args.meta or { }) // {
+      # See comment in postgresql's generic.nix doInstallCheck section
+      broken = stdenv.hostPlatform.isDarwin;
```

Comment:
Doesn't this override `meta.broken` of each package?

Reply:
Yep, that LGTM. That way specific packages can still opt-out from being marked as broken on darwin.
If you do that, feel free to mark `pgvecto-rs` as broken on darwin in this PR, since it already has a `broken` attribute.

Reply:
Sure, that's fine by me as well :)